addons/stock_account/models/account_move.py

```diff
@@ -133,7 +133,7 @@ def _stock_account_prepare_anglo_saxon_out_lines_vals(self):
 
                 # Compute accounting fields.
                 sign = -1 if move.type == 'out_refund' else 1
-                price_unit = line._stock_account_get_anglo_saxon_price_unit()
+                price_unit = line.with_context(force_company=self.company_id.id)._stock_account_get_anglo_saxon_price_unit()
```

Comment:
IMO, you shouldn't be force to do such thing. For me, there is probably a bad managment of the company somewhere inside `_stock_account_get_anglo_saxon_price_unit`

Reply:
summary of discussions:
`account.move.line` has company_id set on it and thus, it should be company safe by itself.
There are some calls made inside the method or one of the method that override it that aren't company safe.
It's there that it should be fixed.